packages/circuit-ui/components/Carousel/__snapshots__/Carousel.spec.js.snap

```diff
@@ -199,6 +199,7 @@ exports[`Carousel styles should render with children as a function 1`] = `
 
 .circuit-37 {
   font-weight: 400;
+  margin-bottom: 16px;
```

Comment:
Hm, should the carousel components also be. given the noMargin prop? Was this an unintended visual change?

Reply:
As per Chromatic, it looks like it's not actually changing anything

Reply:
The margin is reset a few lines down.